packages/calcite-components/src/components/functional/FilteredItems.tsx

```diff
@@ -0,0 +1,16 @@
+import { FunctionalComponent, h } from "@stencil/core";
+
+interface ExpandToggleProps {
+  filterEnabled: boolean;
+  items: string[];
+}
+
+export const ExpandToggle: FunctionalComponent<ExpandToggleProps> = ({ filterEnabled, items }) => {
```

Comment:
@anveshmekala @jcfranco is there any mechanism for fectching some common message strings from a functional component?

If not, I can just pass in messages from the list component but that kind of defeats the benefit of the functional component and I would just have this in the List component itself.

Reply:
The strings would be pretty generic like `"Filter enabled"`, `"Displaying {num} Items"`, `"Item {x}: My Item"`.